presto-spi/src/test/java/com/facebook/presto/spi/type/TestTypeSignature.java

```diff
@@ -151,7 +159,7 @@ private static void assertSignature(
         for (int i = 0; i < signature.getParameters().size(); i++) {
             assertEquals(signature.getParameters().get(i).toString(), parameters.get(i));
         }
-        assertEquals(signature.toString(), expectedTypeName);
+        assertEqualsIgnoreCase(signature.toString(), expectedTypeName);
```

Comment:
If we support case sensitive field names in ROWs, will this check still be correct?